src/test/scala/org/broadinstitute/dsde/workbench/leonardo/TestSupport.scala

```diff
@@ -0,0 +1,11 @@
+package org.broadinstitute.dsde.workbench.leonardo
+
+import scala.concurrent.{Await, Awaitable}
+import scala.concurrent.duration.Duration
+
+/**
+  * Created by dvoet on 6/27/17.
+  */
+trait TestSupport {
```

Comment:
I'm wondering if we should prefer built-in scalatest functionality for verifying Futures instead of `Await.result` calls. See: http://doc.scalatest.org/3.0.0/index.html#org.scalatest.concurrent.Futures

Example:
```
whenReady(future) { s =>
  s should be ("hello")
}
```